bundles/org.openhab.core.config.discovery.mdns/src/main/java/org/openhab/core/config/discovery/mdns/internal/MDNSDiscoveryService.java

```diff
@@ -232,4 +253,14 @@ private void considerService(ServiceEvent serviceEvent) {
             }
         }
     }
+
+    /*
```

Comment:
javadoc missing character:
```suggestion
    /**
```

Reply:
Ok.